src/util.js

```diff
@@ -14,7 +14,7 @@
  */
 var Util = window.Util || {};
 
-Util.MIN_TIMESTEP = 0.001;
+Util.MIN_TIMESTEP = 0.0001;
```

Comment:
Boris,
I did take a look at this and it seems that FF returns smaller deltas.   Not sure why this is the case.   

I whipped together a page to monitor the differences: https://github.com/caseyyee/DevicemotionTest